### ISSUE TITLE
Precision & Recall on test set

### ISSUE THREAD

**Issue by mahavird** [chenyuntc/simple-faster-rcnn-pytorch#104]
Can you kindly tell me how I can evaluate precision and recall on my validation set.
